### PR TITLE
Adding Links and hover effect to "Services", "Repairs" and "Contact" headings in the footer section

### PR DESCRIPTION
**Pull request by akanksha984**
## Summary
I have added Links and hover effect to "Services", "Repairs" and "Contact" headings in the footer section.
Tech stack used: HTML, CSS

## Type of change
_Please select the relevant option._
- [✓] Bug fix
- [✓] New feature
- [ ] Breaking change
- [ ] Documentation update
- [ ] Other (please describe)

## Checklist
_Please tick the relevant options._
- [✓] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [✓] All new and existing tests passed.

## Screenshots

image
image

## Additional context
_Add any other context about the pull request here._

### CODE REVIEW

**Comment by kartikayasija** [comment]
@akanksha984 Reduce line height, give gradient color to underline, and apply this to all the pages.

**Comment by akanksha984** [comment]
Done the changes please check @kartikayasija

File: .github/workflows/static.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      - name: Setup Pages
+        uses: actions/configure-pages@v3
+      - name: Upload artifact
+        uses: actions/upload-pages-artifact@v1
+        with:
+          # Upload entire repository
+          path: '.'
+      - name: Deploy to GitHub Pages
+        id: deployment
+        uses: actions/deploy-pages@v2
```
Comment (kartikayasija) [inline .github/workflows/static.yml:43]: Looks good, just remove this file then good to go. Also try not to make unnecessary commits.

Comment (akanksha984) [reply .github/workflows/static.yml]: Done sir. Sorry sir I was just checking it using github pages

**Pull request merged**